airflow/example_dags/example_params_ui_tutorial.py

```diff
@@ -208,6 +216,18 @@
             title="JSON entry field",
             section="Special advanced stuff with form fields",
         ),
+        "array_of_objects": Param(
+            [{"name": "account_name", "country": "country_name"}],
+            "Array with complex objects and validation rules",
```

Comment:
A link to JSON Schema specs might be useful here as well.
```suggestion
            "Array with complex objects and validation rules. "
            "See <a href='https://json-schema.org/understanding-json-schema"
            "/reference/array.html#items'>JSON Schema validation options in specs.</a>",
```